dist/index.html

```diff
@@ -3,9 +3,31 @@
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
-<script defer src="main.bundle.js"></script></head>
+    <!-- <script src="index.js" defer></script> -->
+    <title>Capstone 2</title>
```

Comment:
Lets try to concentrate on removing this inside the body don't you think its when the code will look smart